doc/modules/linear_model.rst

```diff
@@ -465,10 +470,10 @@ of including variables at each step, the estimated parameters are
 increased in a direction equiangular to each one's correlations with
 the residual.
 
-Instead of giving a vector result, the LARS solution consists of a
-curve denoting the solution for each value of the L1 norm of the
+Instead of giving a vector result, the LARS solution consists of a curve
+denoting the solution for each value of the :math:`\ell_1` norm of the
```

Comment:
If you want to do this, please ``git grep L1`` and ``git grep L2`` and make sure that they are all consistent. I think we should go at least for lower case, though in the docstrings I'd prefer ``l1`` over using latex.